### PR TITLE
revert: custom interface

### PR DESCRIPTION
**Pull request by pheiduck**
Reverts #700

### CODE REVIEW

File: src/lib/WireGuard.js [hunk 1]
```diff
@@ -60,18 +59,18 @@ module.exports = class WireGuard {
...
+        await Util.exec('wg-quick down wg0').catch(() => { });
+        await Util.exec('wg-quick up wg0').catch((err) => {
+          if (err && err.message && err.message.includes('Cannot find device "wg0"')) {
+            throw new Error('WireGuard exited with the error: Cannot find device "wg0"\nThis usually means that your host\'s kernel does not support WireGuard!');
           }
 
           throw err;
         });
-        // await Util.exec(`iptables -t nat -A POSTROUTING -s ${WG_DEFAULT_ADDRESS.replace('x', '0')}/24 -o ' + WG_DEVICE + ' -j MASQUERADE`);
+        // await Util.exec(`iptables -t nat -A POSTROUTING -s ${WG_DEFAULT_ADDRESS.replace('x', '0')}/24 -o eth0 -j MASQUERADE`);
```
Comment (peterlewis) [inline src/lib/WireGuard.js:70]: Unsubstantiated observation, but I wonder whether or not this was omitted from the original PR where the ability for changing interface was introduced. Or is this similar to the hardcoded ports, i.e. must always be set to `eth0`?

**Review by peterlewis — COMMENTED** [review]
Looks good. Just a wee outstanding query re reversion to hardcoded `eth0`

File: src/lib/WireGuard.js (hunk 1 above)
Comment (pheiduck) [reply src/lib/WireGuard.js]: Done: https://github.com/wg-easy/wg-easy/pull/714/commits/d16ed82290050fbf383f2a63b5a5ee8fe6337cf6

**Review by peterlewis — APPROVED** [review]
Looks good. Before we push this to `production` I would be keen to be enlightened by @WeeJeWel as to its omission up until now...

**Pull request merged**